data/strings/strings.txt

```diff
@@ -11369,7 +11371,7 @@
 
   [login]
     tags = android,ios
-    en = Log In
+    en = OpenStreetMap Login
```

Comment:
Does it fit now to the screen? Does it look good or duplicate another OpenStreetMap string on the same screen?

Reply:
Given the screenshot, I think we definitely should use the "OpenStreetMap Login".  From a UI viewpoint I think it's important that the biggest title text immediately gives people the right impression.